src/app/workspaces/workspace-details/devfile/workspace-devfile-editor.controller.ts

```diff
@@ -125,7 +129,38 @@ export class WorkspaceDevfileEditorController {
     this.devfileYaml = this.workspaceDevfile ? jsyaml.safeDump(this.workspaceDevfile) : '';
     this.devfileDocsUrl = this.cheBranding.getDocs().devfile;
     const yamlService = (window as any).yamlService;
-    this.cheAPI.getDevfile().fetchDevfileSchema().then(jsonSchema => {
+    this.cheAPI.getDevfile().fetchDevfileSchema().then(async jsonSchema => {
```

Comment:
Could you please use either promise chains or async/await just not to mix them up?